.gitignore

```diff
@@ -71,11 +72,8 @@ $RECYCLE.BIN/
 
 # Windows shortcuts
 *.lnk
-<<<<<<< HEAD
 /trainSMART-android.iml
 /.idea/workspace.xml
-=======
 .idea/compiler.xml
```

Comment:
You can drop all the idea entries since you added  the entire dir to ignore

Reply:
Oh, you are right. Thanks!